presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -169,12 +165,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                 new IterativeOptimizer(
                         stats,
                         ImmutableList.of(new CanonicalizeExpressions()),
-                        ImmutableSet.of(
-                                new CanonicalizeJoinExpressions(),
-                                new CanonicalizeProjectExpressions(),
-                                new CanonicalizeFilterExpressions(),
-                                new CanonicalizeTableScanExpressions()
-                        )
+                        new com.facebook.presto.sql.planner.iterative.rule.CanonicalizeExpressions().rules()
```

Comment:
please import this one, and make the other one to be have a ful import path as that one is deprecated